src/main/java/org/generationcp/middleware/service/impl/inventory/PlantingPreparationDTO.java

```diff
@@ -97,6 +97,16 @@ public void setUnitId(final Integer unitId) {
 		private String designation;
 		private Map<String, StockDTO> stockByStockId = new LinkedHashMap<>();
 		private List<ObservationUnitDTO> observationUnits = new ArrayList<>();
+		private Map<Integer, VariableDTO> entryDetailByVariableId = new LinkedHashMap<>();
```

Comment:
use Map<Integer, Double> because VariableDTO is not needed. valueByEntryDetailId
@cuenyad

Reply:
done.